pkg/security/registry/podsecuritypolicyreview/rest.go

```diff
@@ -70,31 +69,17 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 			errs = append(errs, fmt.Errorf("unable to find SecurityContextConstraints for ServiceAccount %s: %v", sa.Name, err))
 			continue
 		}
-		oscc.DeduplicateSecurityContextConstraints(saConstraints)
-		sort.Sort(oscc.ByPriority(saConstraints))
-		var namespace *kapi.Namespace
-		for _, constraint := range saConstraints {
-			var (
-				provider kscc.SecurityContextConstraintsProvider
-				err      error
-			)
-			pspsrs := securityapi.PodSecurityPolicySubjectReviewStatus{}
-			if provider, namespace, err = oscc.CreateProviderFromConstraint(ns, namespace, constraint, r.client); err != nil {
-				errs = append(errs, fmt.Errorf("unable to create provider for service account %s: %v", sa.Name, err))
-				continue
-			}
-			_, err = podsecuritypolicysubjectreview.FillPodSecurityPolicySubjectReviewStatus(&pspsrs, provider, pspr.Spec.Template.Spec, constraint)
-			if err != nil {
-				glog.Errorf("unable to fill PodSecurityPolicyReviewStatus from constraint %v", err)
-				continue
-			}
-			sapsprs := securityapi.ServiceAccountPodSecurityPolicyReviewStatus{pspsrs, sa.Name}
-			newStatus.AllowedServiceAccounts = append(newStatus.AllowedServiceAccounts, sapsprs)
+		assigner := newSCCAssigner(&newStatus, pspr.Spec.Template.Spec, sa.Name)
+		err = oscc.AssignConstraints(r.sccMatcher, saConstraints, ns, r.client, assigner)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("unable to assign SecurityContextConstraints for ServiceAccount %s: %v", sa.Name, err))
+			continue
 		}
 	}
-	if len(errs) > 0 {
-		return nil, kapierrors.NewBadRequest(fmt.Sprintf("%s", kerrors.NewAggregate(errs)))
+	for err := range errs {
```

Comment:
That will only print indexes of errors. If anything it would be better to do:
```
if len(errs) > 0 {
    glog.V(4).Infof("PodSecurityPolicyReview err: %v", kerrors.NewAggergate(errs)
}
```